src/network/message_filter.rs

```diff
@@ -0,0 +1,76 @@
+//!
+//! BIP157  Client Side Block Filtering network messages
+//!
```

Comment:
nit: maybe add a comment a la message_blockdata: "This module describes network messages which are used for passing block filters and filter headers inside Bitcoin P2P layer"